submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/security/DeletePermissionTest.kt

```diff
@@ -153,6 +151,39 @@ class DeletePermissionTest(
         assertThat(submissionRepository.getExtByAccNo("DeleteAcc6-2")).isNotNull
     }
 
+    @Test
+    fun `1-7 delete public with regular user and tag access permission`() {
+        val submission = tsv {
+            line("Submission", "DeleteAcc7")
+            line("Title", "Simple Submission")
+            line("AttachTo", "ACollection")
+            line("ReleaseDate", "2018-09-21")
+            line()
+        }.toString()
+
+        assertThat(superUserWebClient.submitSingle(submission, TSV)).isSuccessful()
+        superUserWebClient.givePermissionToUser(RegularUser.email, "ACollection", DELETE.name)
+        assertThrows(WebClientException::class.java) { regularUserWebClient.deleteSubmission("DeleteAcc7") }
```

Comment:
given this will be propaged to user I think it worth to check actual error message is meanfull for them.